test/web_test.rb

```diff
@@ -83,6 +83,20 @@ def job_params(job, score)
       assert_match(/WebJob/, last_response.body)
     end
 
+    it "can quiet all processes" do
```

Comment:
This doesn't really test your change, yes? Were you including it for coverage?

Reply:
I think I understand. 👍🏼

Reply:
(moving my reply inline:) Yeah for coverage and to make sure the functionality still worked when the
input value changed from "1" to "Quiet All"